storage/storage.cpp

```diff
@@ -1269,6 +1280,25 @@ bool Storage::HasLatestVersion(CountryId const & countryId) const
   return CountryStatusEx(countryId) == Status::OnDisk;
 }
 
+bool Storage::IsAllowedToEditVersion(CountryId const & countryId) const
+{
+  auto const status = CountryStatusEx(countryId);
+  switch (status) 
+  {
+    case Status::OnDisk: return true;
```

Comment:
If editing should be disabled in case of a very old, not updated app version, then 6 months period check should also be present here.